plugin/src/main/java/org/opensearch/ml/action/models/DeleteModelTransportAction.java

```diff
@@ -274,7 +274,13 @@ private void deleteController(String modelId) {
             } else {
                 log.warn("The deletion of model controller for model {} returned with result: {}", modelId, deleteResponse.getResult());
             }
-        }, e -> log.error("Failed to re-deploy the model controller for model: " + modelId, e)));
+        }, e -> {
+            if (e instanceof IndexNotFoundException) {
+                log.warn("Model controller not deleted due to no model controller found for model: " + modelId);
```

Comment:
This is not warn level, could be debug level

Reply:
Resolved